### PR TITLE
Modify tests related to edit function to include new birthday field

### PR DESCRIPTION
**Pull request by appleraincoat**

### CODE REVIEW

**Review by UdhayaShan1 — APPROVED** [review]
LGTM, thanks

File: src/test/java/seedu/realodex/logic/parser/EditCommandParserTest.java [hunk 1]
```diff
@@ -222,6 +232,12 @@ public void parse_oneFieldSpecified_success() {
         descriptor = new EditPersonDescriptorBuilder().withTags(VALID_TAG_BOB, VALID_TAG_AMY).build();
         expectedCommand = new EditCommand(targetIndex, descriptor);
         assertParseSuccess(parser, userInput, expectedCommand);
+
+        // birthday
+        userInput = targetIndex.getOneBased() + BIRTHDAY_DESC_AMY;
+        descriptor = new EditPersonDescriptorBuilder().withBirthday(VALID_BIRTHDAY_AMY).build();
+        expectedCommand = new EditCommand(targetIndex, descriptor);
+        assertParseSuccess(parser, userInput, expectedCommand);
```
Comment (jyztintan) [inline src/test/java/seedu/realodex/logic/parser/EditCommandParserTest.java:240]: great

File: src/test/java/seedu/realodex/testutil/EditPersonDescriptorBuilder.java [hunk 2]
```diff
@@ -113,6 +114,15 @@ public EditPersonDescriptorBuilder withRemark(String remark) {
         return this;
     }
 
+    /**
+     * Sets the {@code Birthday} of the {@code EditPersonDescriptor} that we are building.
+     */
+    public EditPersonDescriptorBuilder withBirthday(String birthday) {
```
Comment (jyztintan) [inline src/test/java/seedu/realodex/testutil/EditPersonDescriptorBuilder.java:120]: good

**Review by jyztintan — APPROVED** [review]
Thanks for adding, this ensures the integrity of our edit functionality to birthday field.

**Pull request merged**